### PR TITLE
Translated title of content

### LINKED ISSUE #116: Problems with R-novice-gapminder table of contents and headers
There are several small things that need to be fixed in the latest rendered [R-novice-gapminder translation](https://swcarpentry-ja.github.io/r-novice-gapminder/ja/index/index.html).

In the table of contents:
- Lesson 5 title is in English
- Lesson 8 is empty
- Lesson 13 title is in in English
- The translation for "finish" as 仕上がり doesn't really make sense in this context (consider changing in [lang.md](https://github.com/swcarpentry-ja/i18n/blob/ja/lang.md)). Probably 終わり is better?

In the outline for each lesson, "Teaching" still shows up as　教えります (should be 講義, as set in `lang.md`)

### PR DESCRIPTION
**Pull request by masamiy**
There are conflicts and Joel and Masami hope Tom can fix them!!

### CODE REVIEW

**Comment by TomKellyGenetics** [comment]
Regarding this conflict.

```
+15490 po/r-novice-gapminder.ja.po
<<<<<<< HEAD
"title: データの出力\n"
"講義: 10\n"
"練習: 10\n"
"質問:\n"
"- \"Rで作った図を保存するにはどうしますか？\"\n"
"目標:\n"
"- \"R から図やデータを 出力出来るようになりましょう。\"\n"
"要点:\n"
"- \"RStudio のエクスポートボタンを使って図を保存しましょう。\"\n"
"- \"`write.table` を使って表形式のデータを保存しましょう。\"\n"
=======
"title: データの書き込み\n"
"講義: 10 分\n"
"演習: 10 分\n"
"質問:\n"
"- \"R で作成したプロットやデータを保存するにはどうすればよいですか?\"\n"
"目標:\n"
"- \"R からプロットやデータを書き出せるようになりましょう。\"\n"
"キーポイント:\n"
"- \"'Export' ボタンを使って RStudio からプロットの保存が出来る。\"\n"
"- \"`write.table` を使って表形式のデータの保存が出来る。\"\n"
>>>>>>> cc00d4d8e985e1fc2c0d6303037f8a982a98e70a
```

Conflict appears to have been merged [here](https://github.com/TomKellyGenetics/i18n/commit/7e56a535b20ea2b868c404c64657ada95509ad88):

Screen Shot 2020-08-29 at 17 41 40

Previous commit doesn't have this so it seems to have merged from another branch.

Screen Shot 2020-08-29 at 17 45 36

Related PR: https://github.com/swcarpentry-ja/i18n/pull/50

@atusy seems to have translated this part while doing R lesson 11 - writing data (his copy is the 1st one -- I'll leave it here to consider it in review). I'm going to merge conflicts with @masamiy's version for consistency with other lessons. We have discussed some of these terms before [here](https://github.com/swcarpentry-ja/i18n/issues/41).

**Comment by TomKellyGenetics** [comment]
@masamiy if you "merge" this pull request, it will resolved conflicts in your branch: https://github.com/masamiy/i18n/pull/1

File: po/r-novice-gapminder.ja.po [hunk 1]
```diff
@@ -828,30 +828,30 @@ msgstr ""
...
-"- \"Call functions\"\n"
-"- \"Manage packages\"\n"
-"keypoints:\n"
-"- \"Use RStudio to write and run R programs.\"\n"
-"- \"R has the usual arithmetic operators and mathematical functions.\"\n"
-"- \"Use `<-` to assign values to variables.\"\n"
-"- \"Use `ls()` to list the variables in a program.\"\n"
-"- \"Use `rm()` to delete objects in a program.\"\n"
-"- \"Use `install.packages()` to install packages (libraries).\"\n"
+"レッスンタイトル: \"R と RStudio\"\n"
```
Comment (TomKellyGenetics) [inline po/r-novice-gapminder.ja.po]: For consistency.
```suggestion
"タイトル: \"R と RStudio\"\n"
```

Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po]: Actually, remove this as resolved here: https://github.com/masamiy/i18n/pull/1/commits/9cd9432b193f403444f1422563d4269258e2eeb9

**Comment by TomKellyGenetics** [comment]
As mentioned before, I think we don't need to change repeated words `teaching:`, `exercises:`, `objectives:` in the lesson template. Doing this once in [lang.md](https://github.com/swcarpentry-ja/i18n/blob/ja/lang.md) allows `po4gitbook` to do this automatically. Please change this file instead.

@masamiy I've done this for you: https://github.com/masamiy/i18n/pull/1
https://github.com/masamiy/i18n/pull/1/commits/9cd9432b193f403444f1422563d4269258e2eeb9

This also updates `lang.md` to be consistent with Masami's translations. We can discuss this changes in this PR once they're merged into Masami's branch, they should appear here. https://github.com/swcarpentry-ja/i18n/pull/114/files

**Comment by TomKellyGenetics** [comment]
See the preview for @atusy's translations.

https://swcarpentry-ja.github.io/r-novice-gapminder/ja/_episodes/10-functions/index.html
https://swcarpentry-ja.github.io/r-novice-gapminder/ja/_episodes/11-writing-data/index.html

If you change this template terms in English, the lesson won't display correctly. This commit should correct this: https://github.com/masamiy/i18n/commit/9cd9432b193f403444f1422563d4269258e2eeb9

File: po/r-novice-gapminder.ja.po [hunk 2]
```diff
@@ -828,30 +828,30 @@ msgstr ""
 "---\n"
 "# Please do not edit this file directly; it is auto generated.\n"
 "# Instead, please edit 01-rstudio-intro.md in _episodes_rmd/\n"
-"title: \"Introduction to R and RStudio\"\n"
-"teaching: 45\n"
-"exercises: 10\n"
+"title: \"R と RStudio\"\n"
+"teaching: 45分\n"
```
Comment (TomKellyGenetics) [inline po/r-novice-gapminder.ja.po]: I think "分" is not needed here as it's defined in `lang.md` and already displayed in the webpages. https://swcarpentry-ja.github.io/r-novice-gapminder/ja/_episodes/01-rstudio-intro/index.html
```suggestion
"teaching: 45\n"
```
If too much hassle to change them, I can do it via command-line when this PR is merged.

Comment (masamiy) [reply po/r-novice-gapminder.ja.po]: I would be happy to do whatever works for your command-line magic. Shall I just hit 'Commit suggestion' button?

Comment (joelnitta) [reply po/r-novice-gapminder.ja.po]: Yes, that will apply the suggested change

File: lang.md [hunk 3]
```diff
@@ -15,15 +15,16 @@ contribute: "貢献します"
 source: "ソース"
 cite: "引用します"
 contact: "連絡します"
-keypoints: "要点"
+keypoints: "キーポイント"
```
Comment (TomKellyGenetics) [inline lang.md]:
```suggestion
keypoints: "まとめ"
```

File: po/r-novice-gapminder.ja.po [hunk 4]
```diff
@@ -828,30 +828,30 @@ msgstr ""
 "---\n"
 "# Please do not edit this file directly; it is auto generated.\n"
 "# Instead, please edit 01-rstudio-intro.md in _episodes_rmd/\n"
-"title: \"Introduction to R and RStudio\"\n"
+"title: \"R と RStudio\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: Seems to be missing "introduction".
```suggestion
"title: \"R と RStudioの入門\"\n"
```
とかはどうでしょうか？

File: po/r-novice-gapminder.ja.po [hunk 5]
```diff
@@ -3283,18 +3283,18 @@ msgstr ""
...
 "objectives:\n"
-"- Create self-contained projects in RStudio\n"
+"- RStudio のプロジェクトを作成出来るようになりましょう\n"
 "keypoints:\n"
-"- \"Use RStudio to create and manage projects with consistent layout.\"\n"
-"- \"Treat raw data as read-only.\"\n"
-"- \"Treat generated output as disposable.\"\n"
-"- \"Separate function definition and application.\"\n"
+"- \"RStudio でプロジェクトの生成と管理を行う。\"\n"
+"- \"オリジナルデータは読み込み専用とする。\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: "raw data" is a frequently used technical term that should have a consistent translation. We should decide on a translation and add it to the [glossary](https://github.com/swcarpentry-ja/i18n/wiki/Glossary-for-technical-terms). 

オリジナルデータis probably OK, but what about 生データ?

File: po/r-novice-gapminder.ja.po [hunk 6]
```diff
@@ -3283,18 +3283,18 @@ msgstr ""
...
+"- \"生成された出力は使い捨てとする。\"\n"
+"- \"関数定義とアプリケーションは別々にする。\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: I think アプリケーション typically means more like a "computer application" or "computer app", right?
```suggestion
"- \"関数定義と応用は別々にする。\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 7]
```diff
@@ -4416,21 +4416,21 @@ msgstr ""
...
-"teaching: 40\n"
-"exercises: 15\n"
+"title: \"データの構造\"\n"
+"teaching: 40分\n"
+"exercises: 15分\n"
 "questions:\n"
-"- \"How can I read data in R?\"\n"
-"- \"What are the basic data types in R?\"\n"
-"- \"How do I represent categorical information in R?\"\n"
+"- \"How can I read data in R のデータはどう読めばいいですか?\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"- \"Rにデータをどう読み込ませばいいですか?\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 8]
```diff
@@ -4416,21 +4416,21 @@ msgstr ""
...
+"- \"R の基本的なデータタイプにはどのようなものがありますか?\"\n"
+"- \"R のカテゴリー情報はどのように表現しますか ?\"\n"
 "objectives:\n"
-"- \"To be aware of the different types of data.\"\n"
-"- \"To begin exploring data frames, and understand how they are related to vectors, factors and lists.\"\n"
-"- \"To be able to ask questions from R about the type, class, and structure of an object.\"\n"
+"- \"様々なタイプのデータがあることを理解しましょう。\"\n"
+"- \"データフレームとその要素がベクター、ファクターおよびリストとどのように関連しているかを理解しましょう。\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"- \"データフレームの基本を理解し、そして、データフレームがベクター、ファクターおよびリストとどのように関係しているかを理解しましょう。\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 9]
```diff
@@ -7109,26 +7109,26 @@ msgstr ""
...
 "objectives:\n"
-"- \"Add and remove rows or columns.\"\n"
-"- \"Remove rows with `NA` values.\"\n"
-"- \"Append two data frames.\"\n"
-"- \"Understand what a `factor` is.\"\n"
-"- \"Convert a `factor` to a `character` vector and vice versa.\"\n"
-"- \"Display basic properties of data frames including size and class of the columns, names, and first few rows.\"\n"
+"- \"行や列を追加したり削除出来るようになりましょう。\"\n"
+"- \"`NA`が入っている行の削除が出来るようになりましょう。\"\n"
+"- \"2 つのデータフレームをマージできるようになりましょう。\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"- \"2つのデータフレームをマージできるようになりましょう。\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 10]
```diff
@@ -7109,26 +7109,26 @@ msgstr ""
...
+"- \"`factor （ファクター）`について理解しましょう。\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"- \"`factor`（ファクター）について理解しましょう。\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 11]
```diff
@@ -7109,26 +7109,26 @@ msgstr ""
...
+"- \"`factor` を `character （文字型）`に変換、またはその逆の変換が出来るようになりましょう。\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"- \"`factor` を `character`（文字型）に変換、またはその逆の変換が出来るようになりましょう。\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 12]
```diff
@@ -12294,17 +12294,17 @@ msgstr ""
 "---\n"
 "# Please do not edit this file directly; it is auto generated.\n"
 "# Instead, please edit 08-plot-ggplot2.md in _episodes_rmd/\n"
-"タイトル: ggplot2を用いた出版品質のグラフィックス作成\n"
-"講義： 60\n"
-"練習： 20\n"
+"title: ggplot2 を用いたパブリッシュ出来るグラフィックスの作成\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"title: ggplot2 を用いたパブリッシュ出来る品質のグラフィックスの作成\n"
```

File: po/r-novice-gapminder.ja.po [hunk 13]
```diff
@@ -15895,15 +15895,15 @@ msgstr ""
 "---\n"
 "# Please do not edit this file directly; it is auto generated.\n"
 "# Instead, please edit 12-plyr.md in _episodes_rmd/\n"
-"title: Splitting and Combining Data Frames with plyr\n"
-"teaching: 30\n"
-"exercises: 30\n"
+"title: Splitting and Combining Data Frames with plyr によるデータの分離と併合\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"title: plyr によるデータフレームの分離と併合\n"
```

File: po/r-novice-gapminder.ja.po [hunk 14]
```diff
@@ -16818,20 +16818,20 @@ msgstr ""
...
+"teaching: 40 分\n"
+"exercises: 15 分\n"
 "questions:\n"
-"- \"How can I manipulate dataframes without repeating myself?\"\n"
+"- \"同じ作業を繰り返すことなく、データフレームを操作するにはどうすればよいです?\"\n"
 "objectives:\n"
-"- \" To be able to use the six main dataframe manipulation 'verbs' with pipes in  `dplyr`.\"\n"
-"- \" To understand how `group_by()` and `summarize()` can be combined to summarize datasets.\"\n"
-"- \" Be able to analyze a subset of data using logical filtering.\"\n"
+"- \"`dplyr` にある 6 つの操作 を パイプ を使用して出来るようになりましょう。.\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: I'm not sure why there are extra spaces, I think they should be removed.
```suggestion
"- \"`dplyr` にある6つの操作をパイプを使用して出来るようになりましょう。.\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 15]
```diff
@@ -19935,21 +19935,21 @@ msgstr ""
 "---\n"
 "# Please do not edit this file directly; it is auto generated.\n"
 "# Instead, please edit 16-wrap-up.md in _episodes_rmd/\n"
-"title: Writing Good Software\n"
+"title: 良いソフトウェアの作成\n"
 "teaching: 15\n"
 "exercises: 0\n"
 "questions:\n"
-"- \"How can I write software that other people can use?\"\n"
+"- \"他の人も使えるソフトウェアを描くにはどうしたらよいですか?\"\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"- \"他の人も使えるソフトウェアを書くにはどうしたらよいですか?\"\n"
```

**Review by joelnitta — COMMENTED** [review]
Thanks!
You don't need to add 分 after the minutes (e.g., 45 not 45分), since it will get added automatically when the website is rendered. @TomKellyGenetics will take those out from this PR though, so don't worry about it.
I noticed you often included a space after Roma-ji (for example, "R は" vs. "Rは"). I think we need to decide as a group what the appropriate style should be, so I didn't suggest to remove them in most cases.

File: po/r-novice-gapminder.ja.po (hunk 4 above)
Comment (masamiy) [reply po/r-novice-gapminder.ja.po:831]: Usually no particle before 入門。So, may be R と Rstudio 入門 ？

File: po/r-novice-gapminder.ja.po (hunk 5 above)
Comment (masamiy) [reply po/r-novice-gapminder.ja.po]: I am sure 生のデータ should be fine, but what about 元のデータ？ In the actual explanation about this topic, it did not use either - They just said 'data'!  So, whatever you are comfortable with :)

File: po/r-novice-gapminder.ja.po (hunk 6 above)
Comment (masamiy) [reply po/r-novice-gapminder.ja.po]: In our translation further down of this section uses 適応、but I feel 適用 works better, so, what about 関数定義と適用は別々にする？

File: po/r-novice-gapminder.ja.po (hunk 4 above)
Comment (masamiy) [reply po/r-novice-gapminder.ja.po]: Without 'の' looks more common - what about RとRStudio入門？

File: po/r-novice-gapminder.ja.po (hunk 6 above)
Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po]: +1 for consistency. Joel is right, this about defining and using functions in different script (not to be confused with アプリ). This has already been translated as a heading in the lesson (quoted below).

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L3539


We should change [this line](https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L3539) to match it.

File: po/r-novice-gapminder.ja.po (hunk 5 above)
Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po]: @rikutakei have you looked into this? Seems like Joel's suggestion is being used in the lessons already.

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L877
https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L3531

File: po/r-novice-gapminder.ja.po (hunk 6 above)
Comment (joelnitta) [reply po/r-novice-gapminder.ja.po]: Actually, 適応 seems to be in error. That would mean more like "adaptation", not "application". They should both be 適用.

File: po/r-novice-gapminder.ja.po [hunk 16]
```diff
@@ -3536,7 +3536,7 @@ msgstr ""
 # header
 #: r-novice-gapminder/_episodes/02-project-intro.md:120
 msgid "### Separate function definition and application"
-msgstr "### 関数の定義と適応を分けましょう"
+msgstr "### 関数定義と適用は別々にしましょう"
```
Comment (TomKellyGenetics) [inline po/r-novice-gapminder.ja.po:3539]: Updated to reflect changes discussed. Just realised I can edit the branch on gituhb.

https://github.com/masamiy/i18n/blob/a106a54f042ebf174b3c26269bde4faf405494c4/po/r-novice-gapminder.ja.po#L63

Didn't realise I could do that. https://github.com/masamiy/i18n/edit/toc-edit/po/r-novice-gapminder.ja.po?pr=%2Fswcarpentry-ja%2Fi18n%2Fpull%2F114

File: po/r-novice-gapminder.ja.po [hunk 17]
```diff
@@ -12279,7 +12279,7 @@ msgid ""
 "exercises: 20\n"
 "questions:\n"
 "- \"How can I create publication-quality graphics in R?\"\n"
-"objectives:\n"
+"questions:\n"
```
Comment (TomKellyGenetics) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"objectives:\n"
```

File: po/r-novice-gapminder.ja.po [hunk 18]
```diff
@@ -12295,11 +12295,11 @@ msgstr ""
...
 "title: ggplot2 を用いたパブリッシュ出来る品質のグラフィックスの作成\n"
-"講義： 60 分\n"
-"練習： 20 分\n"
-"質問：\n"
+"teaching: 60\n"
+"exercises: 20\n"
+"questions:\n"
 "- \"R でパブリッシュ出来る品質のグラフィックを作成するにはどうすればよいですか？\"\n"
-"目標：\n"
+"questions:\n"
```
Comment (TomKellyGenetics) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"objectives:\n"
```

**Comment by TomKellyGenetics** [comment]
This removes 分 for minutes as discussed above (and undoes some mistakes I made in the process of doing that).

File: po/r-novice-gapminder.ja.po (hunk 6 above)
Comment (joelnitta) [reply po/r-novice-gapminder.ja.po]:
```suggestion
"- \"関数定義と適用は別々にする。\"\n"
```

File: po/r-novice-gapminder.ja.po (hunk 5 above)
Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po]:
```suggestion
"- \"生のデータは読み込み専用とする。\"\n"
```

Changed for consistency with other lessons. I prefer "raw" to "original" if this still makes in Japanese. Keeping an unchanged copy of the raw data (in contrast to how most people work in Excel) is an important point in the context of the lesson.

生データ or 加工データ seem to be acceptable computer jargon so we could drop the の particle.
https://jisho.org/search/raw%20data

**Review by TomKellyGenetics — COMMENTED** [review]
Looks like this is almost done! 👍

File: po/r-novice-gapminder.ja.po (hunk 5 above)
Comment (joelnitta) [reply po/r-novice-gapminder.ja.po]: I think 生データ is fine, let's go with that.
```suggestion
"- \"生データは読み込み専用とする。\"\n"
```

**Comment by joelnitta** [comment]
I think we've reviewed this thoroughly enough. Great job team!

**Comment by joelnitta** [comment]
forgot to mention... this resolves issue #111

**Comment by TomKellyGenetics** [comment]
PR #115 seems to have been merged at the same time 👍

**Comment by TomKellyGenetics** [comment]
Lessons updated: https://swcarpentry-ja.github.io/r-novice-gapminder/ja/index/

@joelnitta or @rikutakei should be able to do this by running this in future.

```
sh wrapper.sh --repo r-novice-gapminder --account swcarpentry-ja --webpages
```

**Pull request merged**